### ISSUE TITLE
Loss computation

### ISSUE THREAD

**Issue by wuying39** [ultmaster/neuralpredictor.pytorch#6]
First of all, thank you for the reproduction code. During the training process, I found that the loss does not converge, what is the reason for this? Does it have an effect on the experiment?
